packages/client/modules/email/components/EmailNotifications/EmailNotificationTemplate.tsx

```diff
@@ -0,0 +1,77 @@
+import graphql from 'babel-plugin-relay/macro'
+import {EmailNotificationTemplate_notification$key} from 'parabol-client/__generated__/EmailNotificationTemplate_notification.graphql'
+import React, {ReactNode} from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../../../../styles/paletteV3'
+import relativeDate from '../../../../utils/date/relativeDate'
+import {linkStyle} from '../NotificationSummaryEmail'
+
+const notificationBodyStyle = {paddingTop: 8, paddingBottom: 8} as React.CSSProperties
+
+const rowStyle = {
```

Comment:
+1 you can use `React.CSSProperties` type for all the objects holding CSS styles, you'll get a good type check and nice autocomplete

Reply:
TIL!